docs/alert.html

```diff
@@ -1,165 +1,158 @@
-                >
-                  X
-                </button>
-              </div>
-              <code>
-                <iframe
-                  src="https://carbon.now.sh/embed?bg=rgba%28250%2C250%2C250%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=142%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert%2520alert-primary%2522%253E%250A%2520%2520A%2520primary%2520alert%250A%2520%2520%253Cbutton%250A%2520%2520%2520%2520type%253D%2522button%2522%250A%2520%2520%2520%2520class%253D%2522icon-btn%2520icon-btn-secondary%2522%250A%2520%2520%2520%2520id%253D%2522btn-close-alert%2522%250A%2520%2520%253E%250A%2520%2520%2520%2520X%250A%2520%2520%253C%252Fbutton%253E%250A%253C%252Fdiv%253E"
-                  sandbox="allow-scripts allow-same-origin"
-                  class="h-18 w-full border-none"
-                >
-                </iframe>
-                <iframe
-                  src="https://carbon.now.sh/embed?bg=rgba%28250%2C250%2C250%2C1%29&t=one-light&wt=none&l=javascript&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14.5px&lh=142%25&si=false&es=2x&wm=false&code=const%2520btnAlert%2520%253D%2520document.querySelector%28%27%2523btn-close-alert%27%29%253B%250A%250AbtnAlert.addEventListener%28%27click%27%252C%2520function%2520%28e%29%2520%257B%250A%2520%2520e.currentTarget.parentNode.style.display%2520%253D%2520%27none%27%253B%250A%257D%29%253B"
-                  class="h-14 w-full border-none d-block"
-                  sandbox="allow-scripts allow-same-origin"
-                >
-                </iframe>
-              </code>
-            </div>
-          </article>
-          <div class="d-flex content-between text-8">
-            <a href="/docs/typography.html" class="btn btn-secondary-outlined text-white">Typography</a>
-            <a href="/docs/avatar.html" class="btn btn-secondary-outlined"
-              >Avatar</a
-            >
-        </section>
-      </div>
-    </main>
-    <footer class="h-12 bg-secondary d-flex flex-col items-center">
-      <h5 class="mt-2 text-black text-10">Made by Debarshi Bhattacharjee</h5>
-      <div class="py-1">
-        <a href="https://github.com/Debarshi95">
-          <i class="fab fa-github text-16 text-black"></i>
-        </a>
-        <a href="https://www.linkedin.com/in/debarshi-bhattacharjee-5a9289164">
-          <i class="fab fa-linkedin text-16 text-black"></i>
-        </a>
-        <a href="https://twitter.com/debarshib13">
-          <i class="fab fa-twitter text-16 text-black"></i>
-        </a>
+      <article>
+        <h5 class="my-2">Alert with close button</h5>
+        <div class="section-border">
+          <div class="alert alert-primary mb-2">
+            A primary alert
+            <button type="button" class="icon-btn icon-btn-secondary" id="btn-close-alert">
+              X
```

Comment:
You could have followed the same naming structure that you have used for the alert. like instead of `icon-btn` you can use `btn-icon` using the element first then they type of the btn.